test/rig_tests/proxy/response_from/async_http_test.exs

```diff
@@ -98,4 +98,145 @@ defmodule RigTests.Proxy.ResponseFrom.AsyncHttpTest do
     # ...but the client got the response sent to the HTTP internal endpoint:
     assert Jason.decode!(res_body)["data"] == async_response
   end
+
+  test_with_server "Given response_from is set to http_async and the backend responds with 200, the http response is synchronous." do
```

Comment:
In a similar vein, I would change this to

```elixir
  test_with_server "Given response_from=http_async, when the backend responds with 200, RIG forwards this 200-response (and does not expect an asynchronous response for this request)." do
```

Reply:
done